packages/client/tests/functional/relationMode/tests_1-to-1.ts

```diff
@@ -601,6 +603,7 @@ testMatrix.setupTestSuite(
                           [Providers.MYSQL]: 'Unique constraint failed on the constraint: `ProfileOneToOne_userId_key`',
                           [Providers.SQLSERVER]: 'Unique constraint failed on the constraint: `dbo.ProfileOneToOne`',
                           [Providers.SQLITE]: 'Unique constraint failed on the fields: (`userId`)',
+                          [ProviderFlavors.VITESS_8]: 'Unique constraint failed on the (not available)',
```

Comment:
Btw, notice that Prisma error messages on Vitess are somewhat lacking compared to the other providers.

Reply:
https://github.com/prisma/prisma/issues/10829